packages/@angular/cli/blueprints/ng2/files/__path__/app/app.component.spec.ts

```diff
@@ -18,21 +18,21 @@ describe('AppComponent', () => {
   });
 
   it('should create the app', async(() => {
-    const fixture = TestBed.createComponent(AppComponent);
-    const app = fixture.debugElement.componentInstance;
+    const fixture: ComponentFixture<AppComponent> = TestBed.createComponent(AppComponent);
```

Comment:
For harmonization with [component template](https://github.com/angular/angular-cli/blob/master/packages/@angular/cli/blueprints/component/files/__path__/__name__.component.spec.ts#L10)